README.md

```diff
@@ -8,6 +8,8 @@
 
 For Hue & Friends of Hue remotes checkout [Hue-remotes-HASS](https://github.com/robmarkcole/Hue-remotes-HASS)
 
+**Note:** this repo is now in stability mode, our priority is maintaining existing functionality and minimising work for the maintainers. Please do not open feature request issues or create PR adding new functionality. Only PR to address breaking changes in HA compatibility will be considered.
+
```

Comment:
Perfect, IMO.

Maybe even more explicit towards migration to HA:

Sensor support in the official hue integration is more feature-complete than this one, and if you're already using [Hue-remotes-HASS](https://github.com/robmarkcole/Hue-remotes-HASS), then the refresh rate for sensors in the official integration will be automatically the fastest possible, so there is no real enhancement in using this.

Reply:
That is a good point. Maybe we should go further, and set a date for archiving this repo

Reply:
as long as the sensors created by this custom integration are rock solid and the core sensors go unavailable all the time, because of which ever reason, this custom integration should really be kept alive.... not saying it should be expanded, but please don't inactivate it